internal/drivers/anka/pool.go

```diff
@@ -0,0 +1,237 @@
+package anka
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ankabin = "/usr/local/bin/anka"
```

Comment:
if this never changes it should not be a var

Reply:
done